docs/release-notes/2.42.0.md

```diff
@@ -0,0 +1,135 @@
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## What's new in this release
+
+This release introduces support for Ubuntu 24.04, configurable metrics resolutions per service, improved connection management for PostgreSQL and PMM Agent, and experimental dashboards for PMM self-monitoring and MongoDB. 
+
+The release also brings enhancements to MySQL Query Response Time Details, adds new metrics and labels for PostgreSQL, and addresses several issues related to MongoDB monitoring, upgrades, and Query Analytics and High Availability documentation.
+
+Experimental dashboards in PMM are new or redesigned dashboards that are released as a preview feature for you to test and provide feedback.
+
+Keep in mind that these dashboards are not considered fully stable or complete, and their design, functionality, and metrics may change in future releases based on user feedback and further development.
+
+## Release highlights
+
+## Ubuntu 24.04 Noble Numbat support
+
+PMM now officially supports monitoring of databases running on the recently released Ubuntu 24.04 Noble Numbat operating system.
+
+With this addition, PMM continues to provide comprehensive coverage across a wide range of Linux distributions commonly used in database deployments.
+
+Check the full list of supported platforms on the [Percona Software Support Lifecycle webpage](https://www.percona.com/services/policies/percona-software-support-lifecycle).
+
+For information on installing the PMM Client on Linux, see [Set up PMM Client](../../docs/setting-up/client/index.md#package-manager).
+
+## Configurable metrics resolutions per service
+
+You can now configure metrics resolutions on a per-service basis by setting Low-Resolution (LR), Medium-Resolution (MR), and High-Resolution (HR) settings for each exporter individually.
+
+Customizing the resolution settings for individual services means that you can fine-tune your PMM setup to balance data granularity and resource consumption. This will enable you to:
+
+- allocate resources efficiently by focusing on high-resolution data for key services
+- reduce storage requirements by adjusting resolution for less critical components
+- align your monitoring setup with the specific needs of your environment
+  
+This feature is currently accessible via the PMM API and will be integrated into the user interface in a future release. For more information, see the [**Change Postgres Exporter endpoint**](https://percona-pmm.readme.io/reference/changepostgresexporter) in the API documentation.
```

Comment:
I think an example will be needed somewhere (maybe here) of the API call...I looked at the docs and it wasn't obvious on how to change for a service given the api docs ask for an agentID not serviceID.